README.md

```diff
@@ -223,6 +225,36 @@ app.use(function (err, req, res, next) {
 })
 ```
 
+### Verifying a token
+
+If you need to pass the token in a query string rather than an
+HTTP header, you can use the `req.verifyToken(token)` function
+provided by this middleware. This is useful when you want to
+use this library with an oauth callback.
+
+```js
+var csrf = require('csurf')
+var express = require('express')
+
+var app = express()
+app.use(csrf())
+
+app.get('/oauth_callback', function(req, res) {
+  try {
+    req.verifyToken(req.query.state); // throws
```

Comment:
So I thought more on it, and I think what is getting me is the fact that the new method throws (asserts), which is simply a duplication of what is already provided.

Perhaps a more usable API here would not be to throw, but instead return true/false if the token is valid or not. This has the extra benefit of your example here making much more sense, as you are no loner trying to catch some specific exception and instead just have a if/else. It's also more efficient.